cloudferry_devlab/cloudferry_devlab/tests/testcases/test_group_verification.py

```diff
@@ -13,13 +13,13 @@
 # limitations under the License.
 
 import os
-import re
 import unittest
 
-import yaml
 from nose.plugins.attrib import attr
+import yaml
 
-import devlab.tests.functional_test as functional_test
+from cloudferry_devlab.tests import functional_test
+import pkg_resources
```

Comment:
pkg_resources should be placed in the block with internal libraries (os, unittest)